bode/bode/models/task.py

```diff
@@ -9,14 +11,23 @@
 from bode.models.utc_datetime import UTCDateTime
 
 
+class Status(enum.Enum):
+    todo = "todo"
+    indirectly_done = "indirectly_done"
+    done = "done"
+
+    def __str__(self):
+        return self.name
+
```

Comment:
I think we need the `list()` method for further schema validations:
`@classmethod
    def list(cls):
        return [c.value for c in cls]`